By default 'mutt' embeds configure flags and the output of 'gcc -v',
which contains the store file name of Bash and GCC.  This patch makes
sure we don't embed a reference to these in 'mutt'.

--- mutt-1.5.23/txt2c.sh	2015-06-26 22:56:56.500731643 +0200
+++ mutt-1.5.23/txt2c.sh	2015-06-26 22:57:26.664583900 +0200
@@ -21,6 +21,8 @@ txt2c_fallback () {
 	echo ";"
 }
 
+echo "unsigned char $1[] = \"value of '$1' not kept\";"
+exit 0
 ./txt2c test </dev/null >/dev/null 2>&1 &&
 ./txt2c "$1" ||
 txt2c_fallback "$1"

